projects/plugins/backup/src/js/components/MyPlan.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Internal dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+import {
+	isJetpackBackup,
+	isJetpackBundle,
+	isJetpackLegacyPlan,
+} from '../../../../jetpack/_inc/client/lib/plans/constants';
```

Comment:
@Automattic/jetpack-crew - Can anyone explain why this seems to work?  I've tested it on a JN site without the Jetpack plugin and would expect it to have a problem.  Is there something in the tooling that pulls it in?

Reply:
Webpack just goes, follows that path, opens the file, copies the content into its own bah, transpiles it and sticks it into the final bundle where root paths are no longer a thing basically. Regular webpack behaviour 👍